jestSetup.js

```diff
@@ -2,7 +2,10 @@ jest.mock('./lib/storage');
 jest.mock('./lib/storage/NativeStorage', () => require('./lib/storage/__mocks__'));
 jest.mock('./lib/storage/WebStorage', () => require('./lib/storage/__mocks__'));
 jest.mock('./lib/storage/providers/IDBKeyVal', () => require('./lib/storage/__mocks__'));
+
 jest.mock('react-native-device-info', () => ({getFreeDiskStorage: () => {}}));
 jest.mock('react-native-quick-sqlite', () => ({
     open: () => ({execute: () => {}}),
 }));
+
+jest.useRealTimers();
```

Comment:
Why is this added here? Do you mind adding a code comment to explain?

Reply:
Hanno added that because otherwise setTimeout is not executed unless we call jest.advanceTime(). So to avoid calling that everywhere we used real times.